sound/soc/sof/ipc4-topology.h

```diff
@@ -40,7 +40,8 @@
 #define SOF_IPC4_GAIN_ALL_CHANNELS_MASK 0xffffffff
 #define SOF_IPC4_VOL_ZERO_DB	0x7fffffff
 
-#define ALH_MAX_NUMBER_OF_GTW   1
+#define ALH_MAX_NUMBER_OF_GTW   16
+#define ALH_MULTI_GTW_BASE	0x50
```

Comment:
@bardliao we should add a similar comment here.

Also for future proofing, I think we should make use of the 'group' information that we set in ACPI tables, so that we can identify different gateways with aggregation. It's very very likely that we will have multiple paths that need to be aggregated in the future.

Reply:
> @bardliao we should add a similar comment here.
> 
> Also for future proofing, I think we should make use of the 'group' information that we set in ACPI tables, so that we can identify different gateways with aggregation. It's very very likely that we will have multiple paths that need to be aggregated in the future.

@plbossart I will close this PR and create a new PR as https://github.com/thesofproject/linux/pull/3569 is merged. Let's discuss there.